vendor/dragino/lsn50-v2.js

```diff
@@ -196,4 +196,21 @@ default:
 		  errors: ["unknown FPort"]
 }
 }
-}
\ No newline at end of file
+}
+
+function normalizeUplink(input) {
+  return {
+    data: {
+      air: [
+        {
+          temperature: input.data.TempC_SHT,
+          relativeHumidity: input.data.Hum_SHT,
+        },
+      ],
+      action: {
+        contactState: input.data.Door_status
```

Comment:
Regarding the above comment; you take `input.data.Door_status`. What values can it have? If we just change CLOSE to CLOSED, and leave this unchanged, something is gonna break, right?

Reply:
Right, forgot that, apologies.